specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachine.json

```diff
@@ -3576,6 +3580,36 @@
         }
       },
       "description": "Describes the properties of the last installed patch summary."
+    },
+    "Placement": {
+      "properties": {
+        "zonePlacementPolicy": {
+          "type": "string",
+          "description": "Specifies the policy for resource's placement in availability zone. <br>Supported policies: {'Any'}<br><br>Policy wise behavior:<br>'Any' - An availability zone will be automatically picked by system as part of resource creation.",
+          "enum": [
```

Comment:
lets explore if addition of enum here means any additional support in any component like ARM/CRP?

Reply:
checked with ARM team. this is just an enum for documentation purposes